docker-hub/repos.md

```diff
@@ -25,8 +25,10 @@ When creating a new repository:
 * You can choose to put it in your Docker ID namespace, or in any
   [organization](orgs.md) where you are an [_owner_](orgs.md#the-owners-team).
 * The repository name needs to be unique in that namespace, can be two
-  to 255 characters, and can only contain lowercase letters, numbers, hyphens (`-`),
-  and underscores (`_`).
+  to 255 characters, and can only contain lowercase letters, numbers, hyphens (`-`), and underscores (`_`).
```

Comment:
Not sure if underscores are allowed. @siarheikarneichyk Could you confirm?

Reply:
Underscores are allowed. But repo name can't end with an underscore or a hyphen. So `test_repo` or `test-repo` is fine. But `test_repo_` ore `test-repo-` won't work. Idk if we want to mention that.

Reply:
Thanks! @siarheikarneichyk Updated

Reply:
Thank you @siarheikarneichyk